appveyor.yml

```diff
@@ -12,6 +12,7 @@ branches:
   - dev
   - master
   - next
+  - alrod-sb-sessions
```

Comment:
remove

Reply:
We need it to build "beta" nuget packages.